### PR TITLE
Benelli_Walbro connector mapping improvements

### PR DESCRIPTION
**Pull request by mi-hol**

### CODE REVIEW

File: firmware/config/boards/Benelli_Walbro/connectors/Benelli_Walbro.yaml [hunk 1]
```diff
@@ -230,34 +233,38 @@ pins:
...
-    class: analog_inputs
+    function: 'side stand Sensor Input  (grounded or open circuit)'
+    class: input_ON_OFF
     color: white
 
   - pin: F7
     function: 'Injection warning light (MIL)'
-    class: analog_inputs
-    type: ??
+    class: analog_outputs
```
Comment (rusefillc) [inline firmware/config/boards/Benelli_Walbro/connectors/Benelli_Walbro.yaml:266]: please clarify what is ``analog_outputs``?

see https://github.com/rusefi/rusefi/blob/master/.github/workflows/gen-pinouts.yaml for all official types. MIL is not one of those types. Happy to merge as is just need your awareness.

Comment (mi-hol) [reply firmware/config/boards/Benelli_Walbro/connectors/Benelli_Walbro.yaml:266]: Sorry, can't seem to find "official types" in the referenced file :( 
Had already created https://github.com/chuckwagoncomputing/interactive-pinout/issues/16 to have a documented list for reference.
Any hint, how I would code MIL correctly?

Comment (rusefillc) [reply firmware/config/boards/Benelli_Walbro/connectors/Benelli_Walbro.yaml:266]: @mi-hol please note lines 23 and below

Comment (mi-hol) [reply firmware/config/boards/Benelli_Walbro/connectors/Benelli_Walbro.yaml:266]: Sorry, but even with this hint, I fail to see the list of valid "class" values to fix "class: analog_outputs".

1. "class" is not mentioned in
`        columns: |
          {
          "pin":"Pin Number",
          "ts_name":"TS Name",
          "type":"Type",
          "function":"Typical Function",
          "color":"Pigtail Color"
          }
`
Therefore to fix "class: analog_outputs"  I'm still unclear about "how to code MIL correctly"

2. the list of valid "type" values seems to start at line 44 but is hard to digest for an inexperienced user due to the preceeding "color" label. Maybe a simple comment would avoid this hurdle for now?

image

**Pull request merged**